### PR TITLE
Fixed issue #18491: Theme editor : List of fonts and colors should be sorted alphabetically

### PR DESCRIPTION
**Pull request by gabrieljenik**

### CODE REVIEW

File: application/config/fonts.php [hunk 1]
```diff
@@ -30,16 +30,6 @@
         ),
     ),
 
-    'font-roboto' => array(
```
Comment (Shnoulle) [inline application/config/fonts.php:33]: Adding a comment for instruction when adding a new font ?

File: themes/survey/fruity/config.xml [hunk 2]
```diff
@@ -87,18 +87,18 @@
         <questionbackgroundcolor type="colorpicker" category="Colors" width="4" title="Question background color" options="on|off" optionlabels="Yes|No">#ffffff</questionbackgroundcolor>
         <checkicon type="icon" category="Colors" width="4" title="Check icon">f00c
             <dropdownoptions>
+                <option value="f061"> <i class="fa fa-arrow-right"></i> Arrow right </option>
```
Comment (Shnoulle) [inline themes/survey/fruity/config.xml:90]: Adding a comment here ? About order ? For theme developer.

File: application/config/fonts.php (hunk 1 above)
Comment (Shnoulle) [reply application/config/fonts.php:33]: The order here is needed ? Not related ?

File: themes/survey/fruity/config.xml [hunk 3]
```diff
@@ -661,17 +661,17 @@
          <font type="dropdown" category="Fonts" width="12" title="Fonts" parent="font">noto
             <dropdownoptions>
                 <optgroup  label="User browser">
-                <option class="font-georgia         " value="georgia"         data-font-package="websafe" >Georgia</option>
-                <option class="font-palatino        " value="palatino"        data-font-package="websafe" >Palatino Linotype</option>
-                <option class="font-times_new_roman " value="times_new_roman" data-font-package="websafe" >Times New Roman</option>
                 <option class="font-arial           " value="arial"           data-font-package="websafe" >Arial</option>
```
Comment (Shnoulle) [inline themes/survey/fruity/config.xml:664]: A comment here about order ?

**Review by Shnoulle — APPROVED** [review]
Adding some comment can be good.

File: application/config/fonts.php (hunk 1 above)
Comment (gabrieljenik) [reply application/config/fonts.php:33]: These font's are also shown on the dropdown

Comment (Shnoulle) [reply application/config/fonts.php:33]: Yes, but order is set in XML only ? Or by here too ?
Just to know :)

File: themes/survey/fruity/config.xml [hunk 4]
```diff
@@ -85,20 +85,21 @@
         <bodybackgroundcolor type="colorpicker" category="Colors" width="4" title="Background color" options="on|off" optionlabels="Yes|No">#ffffff</bodybackgroundcolor>
         <fontcolor type="colorpicker" category="Colors" width="4" title="Font color" options="on|off" optionlabels="Yes|No">#444444</fontcolor>
         <questionbackgroundcolor type="colorpicker" category="Colors" width="4" title="Question background color" options="on|off" optionlabels="Yes|No">#ffffff</questionbackgroundcolor>
+        <!--  When adding new items, please keep fonts ordered alphabetically -->
```
Comment (Shnoulle) [inline themes/survey/fruity/config.xml:88]: fonts ? or icon name ;)

File: application/config/fonts.php (hunk 1 above)
Comment (gabrieljenik) [reply application/config/fonts.php:33]: These fonts are core fonts. Different than the fonts listed in theme xml

Comment (Shnoulle) [reply application/config/fonts.php:33]: Thank you

**Pull request merged**